src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,140 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Ui class to handle user interaction.
+ */
+public class Ui {
+    /**
+     * Scanner to read user input.
+     */
+    private Scanner sc;
+
+    /**
+     * Constructor for Ui class.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Display welcome message.
+     */
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+        + "|  _ \\ _   _| | _____ \n"
+        + "| | | | | | | |/ / _ \\\n"
+        + "| |_| | |_| |   <  __/\n"
+        + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+    }
+
+    /**
+     * Display line to separate user input and output.
+     */
+    public void showLine() {
+        System.out.println("----------------------------------------");
+    }
+
+    /**
+     * Display goodbye message.
+     */
+    public void showBye() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
```

Comment:
Does making these 2 strings constants reduce the number of function calls and make the code neater?